src/main/java/core/basesyntax/service/impl/SupplyAction.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ActionHandler;
+
+public class SupplyAction implements ActionHandler {
+    public void apply(String fruit, int quantity) {
```

Comment:
According to the checklist, there should not be an empty line at the beginning of a class or method implementation. Please remove the empty line 6.